### PR TITLE
added necessary css and html to create delete entry link, modal, and …

### PR DESCRIPTION
**Pull request by NoahLett**
…overlay. #4

### CODE REVIEW

**Comment by NoahLett** [comment]
[Untitled_ Nov 6, 2022 11_34 PM.webm](https://user-images.githubusercontent.com/110089434/200252408-61d02865-c99a-48d1-b9c1-6160d6e0c3d4.webm)

File: js/main.js [hunk 1]
```diff
@@ -222,3 +229,44 @@ function assignEdit(event) {
...
+      var toNumber = Number(string);
+      if (toNumber === data.editing.EntryId) {
+        delete $li[x];
+      }
+    }
+  }
+  $modal.className = 'modal hidden';
+  $overlay.className = 'overlay hidden';
+  data.view = 'entries';
+  location.reload();
```
Comment (BrettAlbright) [inline js/main.js]: you don't want to reload the page for this functionality, you just want to remove the old dom element from the DOM using the `remove` method. Then you don't have to refresh.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Almost there, just one update!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**